src/consensus/pbft/pbft_config.h

```diff
@@ -86,8 +87,10 @@ namespace pbft
 
       if (
         info.pending_cmd_callbacks %
-          PbftConfigCcf::max_update_merkle_tree_interval ==
-        0)
+            PbftConfigCcf::max_update_merkle_tree_interval ==
+          0 ||
+        info.pending_cmd_callbacks <
```

Comment:
What are the effects of this change?

Reply:
functionally nothing, but performance wise ~5% for smallbank